ci/cudf_pandas_scripts/third-party-integration/test.sh

```diff
@@ -32,7 +32,7 @@ main() {
     for lib in ${LIBS//,/ }; do
         lib=$(echo "$lib" | tr -d '""')
         echo "Running tests for library $lib"
-        CUDA_VERSION=$(if [ "$lib" = "tensorflow" ]; then echo "11.8"; else echo "${RAPIDS_CUDA_VERSION%.*}"; fi)
+        CUDA_VERSION=$(if [ "$lib" = "tensorflow" ]; then echo "${RAPIDS_CUDA_VERSION%.*}"; else echo "${RAPIDS_CUDA_VERSION%.*}"; fi)
```

Comment:
We can eliminate the special code path for TensorFlow here.
```suggestion
        CUDA_VERSION=${RAPIDS_CUDA_VERSION%.*}
```

Reply:
Yup, that's the plan. I'll let a CI run finish and update this code. I was able to get tf work with cuda-12.x locally.